crates/sparrow-runtime/src/merge/input.rs

```diff
@@ -297,9 +297,12 @@ mod tests {
         (1..max_len)
             .prop_flat_map(|len| (arb_key_triples(len), arb_i64_array(len)))
             .prop_map(|((time, subsort, key_hash), values)| {
-                let schema =
-                    TableSchema::from_data_fields([Field::new("data", DataType::Int64, true)])
-                        .unwrap();
+                let schema = TableSchema::from_data_fields([Arc::new(Field::new(
```

Comment:
I'm surprised you can't call `FieldRef::new(..)` as an alias for `Arc::new(Field::new(..))`.

Reply:
Yep. Not sure why Rust doesn't let you use the aliased name like that.  I think it's because:

1. You can't define a method on `FieldRef`, since that is really a set of types.
2. You could maybe call `FieldRef::new(...)` instead of `Arc::new(...)`, but that only works for the "outer-most thing" in the alias, which maybe could be confusing. In cases like this you may want to "`new`" both parts. Probably was easier to do the straightforward thing of just having you used the underlying / aliased type operations.